iOS/rainbow/Controller/Leaderboard/ParticipantViewController.swift

```diff
@@ -15,7 +15,7 @@ class ParticipantViewController: UIViewController {
     @IBOutlet weak var timeElapsedLabel: UILabel?
     @IBOutlet weak var scavagingSinceLabel: UILabel?
     @IBOutlet weak var progressLabel: UILabel?
-    @IBOutlet weak var progressSlider: UISlider?
+    @IBOutlet weak var button:UIButton?
```

Comment:
We should make this name more expressive. Can we call it `leaderboardButton` instead? Also, please add a space between the right side of the colon and the class type.